index.css

```diff
@@ -381,3 +399,85 @@ form textarea {
   font-size: 15px;
   color: #172b4d;
 }
+
+@media (min-width: 768px) {
+  nav .menu-bar {
+    display: none;
+  }
+
+  nav {
+    justify-content: space-around;
+    gap: 240px;
+  }
```

Comment:
- Nice work you have done so far 👍 , but there are still some small differences between the Figma template and your design 😄. the width of your navbar doesn't exactly match the design. Please try to fix this. Know that replicating the exact specifications gives confidence to recruiters in your ability to design the exact webpage they'll require 🙏 

- Please also note that the background images does not exactly match what is in the figma template, please try to fix that 🙏 
Your project looks like this:
- Please pay attention to the margin difference between your logo and the left border of the site and also the margin difference between the nav links and the right border of the site.
![figma 1 desktop project](https://user-images.githubusercontent.com/73607512/192858087-fcdce8a5-0789-4557-8508-3e13953ec8c5.PNG)

Figma template:
![figma 1 desktop figma](https://user-images.githubusercontent.com/73607512/192858210-4d1713aa-dbe1-4d14-8252-29398dc9beaf.PNG)